src/CHANGES.txt

```diff
@@ -43,7 +43,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Convert remaining uses of insecure/deprecated mktemp method.
     - Clean up some duplications in manpage.  Clarify portion of manpage on Dir and File nodes.
     - Reduce needless list conversions.
-
+    - Another Python regex syntax fix.
```

Comment:
Please be more specific. No one who reads this will know what was fixed.  So something like "fixed regex in python scanner" would be more helpful.